packages/desktop-client/src/components/transactions/TransactionsTable.js

```diff
@@ -918,13 +942,20 @@ const Transaction = memo(function Transaction(props) {
           exposed={focusedField === 'date'}
           value={date}
           valueStyle={valueStyle}
-          formatter={date =>
-            date ? formatDate(parseISO(date), dateFormat) : ''
-          }
           onExpose={name => !isPreview && onEdit(id, name)}
           onUpdate={value => {
             onUpdate('date', value);
           }}
+          unexposedContent={
+            <View innerRef={dragRef}>
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏adding `drag` handler to only the date column makes the functionality pretty hidden. What do you think about adding a drag-handler at the beginning of the row?

For example: this is what gmail used to do


![](http://designingwebinterfaces.com/blog/wp-content/uploads/2009/07/gmail_drag-handle.png)